docs/windows_developer_guide.md

```diff
@@ -82,3 +82,36 @@ windows_create_package.bat
 
 # Test the Deployment
 You can follow the [baremetal deployment guide](deploying_server_baremetal.md) for information how to deploy and use the ovms.zip package.
+
+# Developer Command Prompt
+For building ovms.exe and running ovms_test.exe with manual bazel commands you must setup proper environment variables.
+Run the batch script in new "Developer Command Prompt for VS 2022" terminal:
+```bat
+cd c:\git\model_server
+windows_setupvars.bat
+```
+
+Reuild unit tests:
+```bat
+bazel --output_user_root=c:\opt build --config=windows --action_env OpenVINO_DIR=c:\opt\openvino/runtime/cmake --jobs=%NUMBER_OF_PROCESSORS% --verbose_failures //src:ovms_test 2>&1 | tee win_build_test.log
+```
+
+Download LLMs
+```bat
+%cd%\windows_prepare_llm_models.bat %cd%\src\test\llm_testing
+```
+
+Copy OpenVINO GenAI and tokenizers libs
+```bat
+copy %cd%\bazel-out\x64_windows-opt\bin\external\llm_engine\copy_openvino_genai\openvino_genai\runtime\bin\Release\*.dll %cd%\bazel-bin\src\
```

Comment:
why this step is needed? @atobiszei this copy folder was problematic also for linux recently

Reply:
This is required for ovms_test exeution, this is a new path after fix for ccache.